e2etest/webhookstests/webhooks.go

```diff
@@ -381,3 +384,102 @@ var _ = ginkgo.Describe("Webhooks", func() {
 		})
 	})
 })
+
+var _ = table.DescribeTable("Webhooks namespace validation",
+	func(resources *metallbconfig.ClusterResources) {
+		err := ConfigUpdaterOtherNS.Update(*resources)
+		framework.ExpectError(err)
```

Comment:
I guess this is not necessary given the next row

Reply:
do you mean that `framework.ExpectError` is redundant because of the following `Expect(err.Error()).To(...)`?

Reply:
yes, if err is nil that row will complain

Reply:
@fedepaol ☝🏽 if that was your suggestion, won't `err.Error()` panic if it's `nil` and we remove the `framework.ExpectError` before it?

Reply:
That's right. I misread the string checking line.

I suggest using
`Expect(err).To(MatchError(ContainSubstring(....))`

That will avoid adding the double check